tsl/test/expected/continuous_aggs_refresh.out

```diff
@@ -423,3 +423,37 @@ SELECT * FROM weekly_temp_without_data;
  Sun May 03 17:00:00 2020 PDT |      3 |               19
 (8 rows)
 
+-- This should fail since we do not allow refreshing inside a
+-- transaction, not even as part of CREATE MATERIALIZED VIEW.
+\set ON_ERROR_STOP 0
+DO
```

Comment:
Can't we just do a regular BEGIN--- COMMIT; transaction block instead of creating a function block. At least it would be good to test that case as well.

Reply:
And it will likely change the error message as well

Reply:
Done that as well.